src/Workleap.Extensions.Mongo.Ephemeral/ReusableMongoRunner.cs

```diff
@@ -51,12 +51,6 @@ public void Rent(Guid renter)
                 options.DataDirectory = dataDirectory;
             }
 
-            var additionalArguments = Environment.GetEnvironmentVariable("WORKLEAP_EXTENSIONS_MONGO_EPHEMERAL_ADDITIONALARGUMENTS")?.Trim();
```

Comment:
I suppose `options.AdditionalArguments` doesn't exist anymore in the latest version

Reply:
It does exist, but it became an array. Parsing arguments from a string is hard.

I checked for usage, but there was not a single result. I believe these environment variables were created to help with using a pre-installed MongoDB instance in CI agents, and the arguments were a way to configure the in-memory storage of the Enterprise edition, which never happened.

Now that EphemeralMongo has evolved to download the binaries at runtime, we could revisit them (while keeping in mind that some are still used, like the binary dir path).